crates/runtime/src/databackend/postgres.rs

```diff
@@ -1,5 +1,6 @@
 use std::{collections::HashMap, mem, sync::Arc};
 
+use crate::secrets::Secret;
```

Comment:
You need to do a similar set up for the postgres data connector in `crates/runtime/src/dataconnector/postgres.rs`

Reply:
Update the support for data connector